docs/pages/concepts/offloading/mutating-webhook.md

```diff
@@ -0,0 +1,91 @@
+---
+title: Mutating webhook
```

Comment:
I would be confused by this title. In the end, we're taking about setting the offloading strategy, so I would expect this as title. Then we can say that this is achieved through a mutating webhook, but this would be the implementation, not the main message in the title.